test/extensions/transport_sockets/tls/cert_validator/test_common.h

```diff
@@ -13,6 +13,8 @@ namespace Extensions {
 namespace TransportSockets {
 namespace Tls {
 
+static const uint32_t SSL_VERIFY_DEPTH_DEFAULT = 9;
```

Comment:
Is this used?

Reply:
no, left behind from previous iteration. Thanks!